DataFormats/HGCDigi/bin/BuildFile.xml

```diff
@@ -0,0 +1,3 @@
+<use name="DataFormats/HGCDigi" />
+<bin   name="HGCSampleTest" file="HGCSampleTest.cpp">
```

Comment:
hi @slava77 thanks for your comments.
I gather that moving to /test does not provide an executable which is available in the $PATH.
I found very useful being able to execute at leisure the HGCSampleTest while developing, is there a way to do that ?
Otherwise I'd see good reasons to keep the .cpp in bin.

Reply:
@Dr15Jones @davidlange6 
since this is the first /bin in \*DataFormats, I'm curious if this is accidental or by design
(I didn't find anything against in https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Restrictions_on_the_package_defi

Reply:
@slava77 IMHO this is just a simple test code as we have many others, and might simply go into a test directory (we have many of them in DataFormats). Discussing in the past with @davidlange6 I think /bin was mostly intended for standalone utilities... Anyway I let others to express themselves

Reply:
hello @fabiocos and all
can one get an executable to be run in parallel to developing, if the .cpp moves to the /test directory ?
Perhaps it was me? ,  when I moved this small program to test, no executable appeared in a folder covered by my $PATH .

Reply:
I also tried with the test programs of one of the DetId directories, and could not execute it.

Reply:
> can one get an executable to be run in parallel to developing, if the .cpp moves to the /test directory ?
> Perhaps it was me? , when I moved this small program to test, no executable appeared in a folder covered by my $PATH .

The executables from `test` end up in `$CMSSW_BASE/test/$SCRAM_ARCH` and can be run from there.

Reply:
In addition, executables in `test` can be run via `scram b runtests` and are run in every IB and pull request.